### PR TITLE
[bug 1191968] Restyle product picker page

### PR DESCRIPTION
**Pull request by willkg**
This restyles the product picker page so it's more like SUMO's. It's
not exactly like SUMO's, but it's enough like SUMO's that we can use the
product icons they use. That'll make some things easier.

Further, this adds product descriptions, but those aren't shown at the
moment because we need to l10n-ize them and we're lacking that
infrastructure.

r?

### CODE REVIEW

File: fjord/feedback/static/css/feedback.less [hunk 1]
```diff
@@ -319,51 +319,72 @@ aside {
...
+
+      > li {
+        background: #ccc;
+        border-radius: 6px;
+        box-shadow: 0 2px 4px rgba(0, 0, 0, 0.1);
+        display: inline-block;
+        margin: 0.4rem 0.4rem 0.8rem;
+        position: relative;
+        vertical-align: top;
+        width: 96%;
```
Comment (willkg) [inline fjord/feedback/static/css/feedback.less]: This is an "eyeballed" number. I couldn't figure out how to get things to line up right.

**Comment by willkg** [comment]
@rehandalal Can you look through this for me?

File: fjord/feedback/static/css/feedback.less (hunk 1 above)
Comment (rehandalal) [inline fjord/feedback/static/css/feedback.less]: i would change this to use:
```css
width: calc(100% - 0.8rem);
```

**Comment by rehandalal** [comment]
r+ with that one change

**Comment by willkg** [comment]
^^^ Fixes width. Will wait for Travis and then land.

**Comment by willkg** [comment]
Thank you!

**Pull request merged**